igvc_platform/src/imu/SerialInterface.h

```diff
@@ -95,9 +95,9 @@ class SerialInterface
    *
    * @param str The string to write to the serial port
    */
-  inline void SerialWriteString(const std::string &str)
+  inline void serialWriteString(const std::string &str)
   {
-    this->connection_port->write(str);
+    this->connection_port_->write(str);
```

Comment:
I don't think there's any point in doing `this->connection_port_->write(str)` when you could just do
`connection_port_->write(str)`

Reply:
Yeah, I know. I just wanted to keep it all one style, but I changed it